stylesheets/commons/Brackets.less

```diff
@@ -478,12 +478,12 @@
 	}
 
 	@media ( max-width: 768px ) {
-		bottom: 45px;
-		left: 1px !important;
+		bottom: unset;
```

Comment:
arent't the 45px bottom needed for the bar with trending/tournaments/contribute/search/contents

Reply:
The brkts-popup has a max-height of `65vh` so it should be okay